spring-cloud-alibaba-examples/rocketmq-example/rocketmq-broadcast-example/rocketmq-broadcast-consumer1-example/src/main/resources/application.yml

```diff
@@ -4,9 +4,9 @@ spring:
   application:
     name: rocketmq-broadcast-consumer1-example
   cloud:
+    function:
+      definition: consumer;
     stream:
-      function:
```

Comment:
why changed this?

Reply:
rocketMQ related changes have been restored and submitted in another PR